genai-perf/genai_perf/parser.py

```diff
@@ -35,18 +35,11 @@
 import genai_perf.logging as logging
 import genai_perf.utils as utils
 from genai_perf.constants import (
-    CNN_DAILY_MAIL,
     DEFAULT_ARTIFACT_DIR,
     DEFAULT_COMPARE_DIR,
     DEFAULT_TRITON_METRICS_URL,
-    OPEN_ORCA,
-)
-from genai_perf.inputs.inputs import (
-    Inputs,
-    ModelSelectionStrategy,
-    OutputFormat,
-    PromptSource,
 )
+from genai_perf.inputs import input_constants as ic
```

Comment:
Is there a reason for the inconsistency in how we import constants? This is different than in inputs.py